pallets/manta-pay/src/lib.rs

```diff
@@ -759,7 +766,13 @@ where
         &self,
         output: config::UtxoAccumulatorOutput,
     ) -> Option<Self::ValidUtxoAccumulatorOutput> {
-        if UtxoAccumulatorOutputs::<T>::contains_key(encode(output)) {
+        let accumulator_output = fp_encode(output).ok()?;
+        // Checking for an empty(zeroed) byte array.
+        // This happens for UTXOs with value = 0, for which you dont need
+        // a membership proof, but you still need a root(in this case zeroed).
```

Comment:
I'd like to see a `why` (because zerocoins are created on the fly?) instead of a `what` description here or a link to further documentation

Reply:
However, this is minor and can be added as part of #910